edb/edgeql/parser/__init__.py

```diff
@@ -19,9 +19,14 @@
 
 from __future__ import annotations
 
+from typing import TYPE_CHECKING, List
```

Comment:
Why not just `from typing import *` as everywhere else?

Reply:
Not sure. Probably just general allergy to `*`. Fixed.